src/app/src/components/EmbeddedMapUnauthorized.jsx

```diff
@@ -24,10 +32,12 @@ function EmbeddedMapUnauthorized() {
                 <a href={EmbeddedMapInfoLink}>OAR Embedded Map</a> page on our
                 website for packages and pricing options.
             </Typography>
-            <Typography paragraph>
-                Once you have activated it, your OAR Embedded Map Settings will
-                appear on this tab.
-            </Typography>
+            {isSettings && (
+                <Typography paragraph>
+                    Once you have activated it, your OAR Embedded Map Settings
```

Comment:
The contributor cannot activate embedded map themselves, so we should consider changing this to "Once it has been activated for your account..."

Reply:
I made this change in the interest of getting a release deployed.